db/database.go

```diff
@@ -2239,7 +2245,7 @@ func (dbc *Database) GetDatabaseCollectionWithUser(scopeName, collectionName str
 
 // GetDatabaseCollection returns a collection if one exists, otherwise error.
 func (dbc *DatabaseContext) GetDatabaseCollection(scopeName, collectionName string) (*DatabaseCollection, error) {
-	if base.IsDefaultCollection(scopeName, collectionName) && dbc.onlyDefaultCollection() {
+	if base.IsDefaultCollection(scopeName, collectionName) && dbc.OnlyDefaultCollection() {
```

Comment:
Why do we require OnlyDefaultCollection here?

Reply:
legacy code, i only changed this because I wanted to make `OnlyDefaultCollection` public for `rest`.